config/services.yaml

```diff
@@ -5,6 +5,7 @@ imports:
 parameters:
     env(EVALUATOR_PROJECT_DIR): ''
     env(GITHUB_AUTH): ~
+    env(RELEASE_APP_URL): 'https://api.release.spryker.com'
```

Comment:
We created duplicate consts UPGRADER_RELEASE_APP_URL & RELEASE_APP_URL needs to merge them to one and remove the default value if we keep it in service. ConfigurationProvider::DEFAULT_RELEASE_APP_URL not related to this PR

Reply:
Pls use this one UPGRADER_RELEASE_APP_URL